protocols/bgp/packet/path_attributes.go

```diff
@@ -637,7 +637,7 @@ func (pa *PathAttribute) serializeCommunities(buf *bytes.Buffer) uint16 {
 		return 0
 	}
 
-	coms := pa.Value.(*[]uint32)
+	coms := pa.Value.(*types.Communities)
```

Comment:
I think we should at regression tests to ensure the communities are serialized correctly.

Reply:
Absolutely. But not in this quick fix.